src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: (event.key) });
+    });
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: (event.key) });
+    });
+  }
```

Comment:
DRY your code - callback repeats twice
declare a callback below lifecycle methods and pass it as a second arg in lifecycle methods.

```suggestion
  componentDidMount() {
    document.addEventListener('keyup', this.handlerCallback);
  }

  componentWillUnmount() {
    document.removeEventListener('keyup', this.handlerCallback);
  }
  
  handlerCallback = ....
```